src/main/java/it/pagopa/pn/bff/mappers/CxTypeMapper.java

```diff
@@ -11,8 +11,8 @@ public interface CxTypeMapper {
 
     it.pagopa.pn.bff.generated.openapi.msclient.delivery_recipient.model.CxTypeAuthFleet convertDeliveryRecipientCXType(CxTypeAuthFleet cxType);
 
-    it.pagopa.pn.bff.generated.openapi.msclient.delivery_pa.model.CxTypeAuthFleet convertDeliveryPACXType(CxTypeAuthFleet cxType);
-
+    it.pagopa.pn.bff.generated.openapi.msclient.delivery_b2b_pa.model.CxTypeAuthFleet convertDeliveryB2bPACXType(CxTypeAuthFleet cxType);
+    it.pagopa.pn.bff.generated.openapi.msclient.delivery_web_pa.model.CxTypeAuthFleet convertDeliveryWebPACXType(CxTypeAuthFleet cxType);
```

Comment:
add an empty line between  convertDeliveryB2bPACXType, convertDeliveryWebPACXType and convertApiKeysPACXType